### PR TITLE
Add example calculating W spectra

### PR DESCRIPTION
**Pull request by odstrcilt**
Example for calculating of W spectra from SPRED spectrometer as a function of temperature. 

W_spectra_SPRED

### CODE REVIEW

**Comment by odstrcilt** [comment]
@fsciortino could you merge it, please?

File: aurora/radiation.py [hunk 1]
```diff
@@ -686,17 +685,23 @@ def read_adf15(path, order=1):
...
+
+        header0 = header[0].split()
+        header_dict['lam'] = float(header0[0].strip('A'))
+        num_den = np.int_(header0[-2])
+        num_temp = np.int_(header0[-1])
 
         for item in header[1:]:
-            par, val = item.split("=")
-            header_dict[par.strip().upper()] = val.strip()
+            try:
```
Comment (fsciortino) [inline aurora/radiation.py:699]: @odstrcilt could you please explain why you needed to add this try-except? Please add a comment.

File: aurora/radiation.py [hunk 2]
```diff
@@ -717,12 +722,11 @@ def read_adf15(path, order=1):
...
 
         # interpolate PEC on log dens,temp scales
+         # interpolation of log10(PEC) to avoid issues at low ne or Te
         pec_fun = RectBivariateSpline(
             np.log10(dens),
             np.log10(temp),
-            np.log10(
-                PEC
-            ),  # interpolation of log10(PEC) to avoid issues at low ne or Te
+            np.log10( PEC),
```
Comment (fsciortino) [inline aurora/radiation.py:729]: @odstrcilt why eliminating the comment (and adding an unnecessary space)?

**Comment by fsciortino** [comment]
Hi @odstrcilt , sorry for the slow reaction and thanks for opening the PR. It wasn't yet clear to me if you were done with your commits. 

Before merging, I would like some clarifications for the reasoning behind some changes (see comments above). Thanks!

File: aurora/radiation.py (hunk 2 above)
Comment (odstrcilt) [reply aurora/radiation.py:730]: I have just moved the comment before the function to avoid splitting one line in three.

**Review by fsciortino — APPROVED** [review]

**Pull request merged**